ui/test/unit/clinical/controllers/consultationController.spec.js

```diff
@@ -256,6 +254,18 @@ describe("ConsultationController", function () {
             rootScope = $rootScope;
             controller = $controller;
         });
+        appDescriptor = {
+            formatUrl: function (url) {
+                return url;
+            },
+            getConfigValue: function (value) {
+                if(value !== 'ipdButton') {
```

Comment:
ipdButton?
ipdButton is renamed to adtNavigationConfig right?
If that is the case then how are tests passing with still test depending on ipdButton?